### PR TITLE
Wallet content reorganisation

### LINKED ISSUE #85: Reorganising wallet content
[a] add a link the desktop wallet download instructions page to the main docs.vega.xyz homepage (in similar fashion to "Vega concepts", "API References" and "Tools for using Vega"?
[b] There's an extra step if I click Vega wallet. Perhaps there's some unecessary nesting on this part of the site and CLI Wallet and Desktop Wallet could just live under tools. I am not sure what we get from having the umbrella page
[c] The download info for desktop wallet should live on the top level desktop wallet page
[d] [This page](https://docs.vega.xyz/docs/tools/vega-wallet/concepts) could live under the concepts top level rather than apps and tools

### PR DESCRIPTION
**Pull request by candida-d**
Closes #85 

Adds a bit more content to the wallet overview pages, inserts a new column onto the homepage and takes the wallet concepts out of the tools and into concepts

### CODE REVIEW

**Comment by campbellssource** [comment]
Not sure if this matters/is intentional but just noticed that one of the wallet guides is not in the wallet guides section.
Screenshot 2022-04-29 at 11 19 23

**Comment by candida-d** [comment]
> Not sure if this matters/is intentional but just noticed that one of the wallet guides is not in the wallet guides section. Screenshot 2022-04-29 at 11 19 23

It is intentional, but it's not really obvious why, and I've gone back and forth about if it even makes sense. The Create Wallet instructions frequently change when there's a new version, which is why it's in the "latest" folder, whereas the other guides are evergreen and aren't version dependent. @campbellssource

File: docs/tools/vega-wallet/desktop-app/index.md [hunk 1]
```diff
@@ -3,6 +3,9 @@ title: Desktop Wallet
 hide_title: false
 ---
 
-The Vega Wallet is available to use as a desktop application. It allows you to use your existing Vega wallet, create new wallets, or restore wallets. The desktop app also lets you connect to Vega networks and start Vega dApps through the visual interface.
```
Comment (campbellssource) [inline docs/tools/vega-wallet/desktop-app/index.md:6]: Suggestion (non-blocking): remove this line. I this doesn't need to act like news.

File: docs/tools/vega-wallet/desktop-app/index.md [hunk 2]
```diff
@@ -3,6 +3,9 @@ title: Desktop Wallet
...
+The Vega Wallet is available to use as a desktop application.
```
Comment (campbellssource) [inline docs/tools/vega-wallet/desktop-app/index.md:6]: suggestion (non-blocking): delete this line completely. rationale: Docs are not news/announcements. Particularly given the version selector.

File: docs/tools/vega-wallet/index.md [hunk 3]
```diff
@@ -3,7 +3,9 @@ title: Vega Wallet
 hide_title: false
 ---
 
-Vega Wallet allows you to manage wallets and key pairs, and sign transactions. You can interact with a Vega wallet and its keys through two different apps: 
+Vega Wallet allows you to manage wallets and key pairs, and sign transactions.
```
Comment (campbellssource) [inline docs/tools/vega-wallet/index.md:6]: Suggestion: Vega Wallet allows you to manage wallets and key pairs, sign and **broadcast** transactions.

File: docs/tools/vega-wallet/desktop-app/index.md (hunk 2 above)
Comment (candida-d) [reply docs/tools/vega-wallet/desktop-app/index.md:6]: Interesting ... so the point of that is to help people differentiate between the desktop wallet and the CLI - describing what it's useful for. Do you think it does a bad job of that?

**Comment by campbellssource** [comment]
@candida-d I can't see where src/components/HomepageFeatures.js is being used?

File: docs/tools/vega-wallet/desktop-app/index.md (hunk 2 above)
Comment (campbellssource) [reply docs/tools/vega-wallet/desktop-app/index.md:6]: I think the titles "CLI Wallet app" and "Desktop Wallet app" do a good enough job. But I don't think the line does any harm, just one that I think could be trimmed to cut down.

File: docs/tools/vega-wallet/index.md (hunk 3 above)
Comment (candida-d) [inline docs/tools/vega-wallet/index.md:6]:
```suggestion
Vega Wallet allows you to manage wallets and key pairs, sign and broadcast transactions.
```

**Comment by candida-d** [comment]
> @candida-d I can't see where src/components/HomepageFeatures.js is being used?

https://deploy-preview-88--vega-docusaurus.netlify.app/ @campbellssource

**Review by campbellssource — APPROVED** [review]

**Pull request merged**